packages/contracts/contracts/analytics-utilities/libs/QueryAssetHelpers.sol

```diff
@@ -8,175 +8,101 @@ import { ILendingPoolAddressesProvider } from "../../interfaces/ILendingPoolAddr
     )
         internal
-        returns (AssetData memory assetData)
+        returns (AssetSummary memory assetSummary)
     {
         ILendingPool lendingPool = ILendingPool(
             ILendingPoolAddressesProvider(providerAddr).getLendingPool()
         );
+        IIncentivesController incentivesController = IIncentivesController(
+            ILendingPoolAddressesProvider(providerAddr).getIncentivesController()
+        );
 
         AssetMappings a = AssetMappings(ILendingPoolAddressesProvider(providerAddr).getAssetMappings());
 
-
         DataTypes.ReserveData memory reserve = lendingPool.getReserveData(asset, tranche);
-        assetData.tranche = tranche;
-        assetData.asset = asset;
-        (
-            assetData.ltv,
-            assetData.liquidationThreshold,
-            assetData.liquidationBonus,
-            assetData.decimals,
-            //borrowFactor (not used yet)
-        ) = a.getParams(asset, tranche);
-        assetData.canBeCollateral = reserve.configuration.getCollateralEnabled(asset, a);//assetData.liquidationThreshold != 0;
-        assetData.canBeBorrowed = reserve.configuration.getBorrowingEnabled(asset, a);
-        assetData.oracle = ILendingPoolAddressesProvider(providerAddr).getPriceOracle();
-        assetData.totalSupplied = convertAmountToUsd(assetData.oracle, assetData.asset, IAToken(reserve.aTokenAddress).totalSupply(), assetData.decimals, chainlinkOracleConverter);
-        assetData.totalBorrowed = convertAmountToUsd(assetData.oracle, assetData.asset, IAToken(reserve.variableDebtTokenAddress).totalSupply(), assetData.decimals, chainlinkOracleConverter);
-
-        assetData.totalReserves = convertAmountToUsd(assetData.oracle, assetData.asset, IERC20(asset).balanceOf(reserve.aTokenAddress), assetData.decimals, chainlinkOracleConverter);
-        assetData.totalReservesNative = IERC20(asset).balanceOf(reserve.aTokenAddress);
-
-        assetData.utilization = assetData.totalBorrowed == 0
+        assetSummary.reserveData = reserve;
+        assetSummary.tranche = tranche;
+        assetSummary.asset = asset;
+        assetSummary.decimals = IERC20(asset).decimals();
+
+        assetSummary.canBeCollateral = reserve.configuration.getCollateralEnabled(asset, a);
+        assetSummary.canBeBorrowed = reserve.configuration.getBorrowingEnabled(asset, a);
+        assetSummary.oracle = ILendingPoolAddressesProvider(providerAddr).getPriceOracle();
+        assetSummary.supplyCap = a.getSupplyCap(assetSummary.asset);
+        assetSummary.borrowCap = a.getBorrowCap(assetSummary.asset);
+        assetSummary.adminFee = reserve.configuration.getReserveFactor();
+        assetSummary.platformFee = a.getVMEXReserveFactor(asset);
+
+        assetSummary.totalSupplied = IAToken(reserve.aTokenAddress).totalSupply();
+        assetSummary.totalBorrowed = IAToken(reserve.variableDebtTokenAddress).totalSupply();
+        assetSummary.totalReserves = IERC20(asset).balanceOf(reserve.aTokenAddress);
```

Comment:
intentionally that this doesn't include the amount staked?

Reply:
yes. amount staked is separate value. better for analysis to get raw data